package.json

```diff
@@ -40,17 +40,17 @@
   ],
   "license": "MIT",
   "dependencies": {
-    "eslint": "^4.0.0",
+    "eslint": "^4.7.0",
```

Comment:
`^4.0.0` covers `4.7.0`.

Reply:
Sure. But it doesn't hurt to specify `4.7.0` here does it? `ncu` automatically updates all dependencies. It's just a bit easier to update everything

Reply:
"doesn't hurt" is not enough convincing reason to accept the change.